### DISCUSSION TITLE
Feature: show GPG signed commits.

### DISCUSSION THREAD

**Discussion by Muha0644** [community/community#4926] [Mobile]
I don't know if this is the right place to post this feature request, the GUI is kina confusing.

This feature already exist on GitHub desktop, it would be useful if you could see if a commit was signed with GPG. Right now the app just shows who made the commit and other basic info.

**Comment by eliperkins** [comment]
Hey @Muha0644, you're definitely in the right place. Can I get a little more information from you?

You should be able to see when a commit has been verified via GPG signing by tapping the commit to view commit details, with the `Verified` text in the header:

![signedcommit](https://user-images.githubusercontent.com/1051453/128933321-21918091-dbf0-4034-b565-2560d3ea4253.png)

Unverified commits (those whose GPG signatures don't match public key information that GitHub has) will display as `Unverified`.

Where else would you expect to see commit signing information? What other information would you need to see to unblock your workflow?

**Reply by Muha0644** [reply]
That's strange, so it only seems to work on iOS (i'm guessing the image you sent is from iOS)... On the android app it doesn't show this.
On the website it shows that the commit is verified (I know I didn't mess up GPG or something), but on the android app it doesn't show anything about the verification. 

Desktop website:
![Screenshot_20210810_234414](https://user-images.githubusercontent.com/41068336/128939105-509d5dfc-381c-4a10-9ec3-33444f902996.png)

Android App:
![1628631912575](https://user-images.githubusercontent.com/41068336/128939788-33b9648f-9719-427d-b928-ec3957d6ba35.jpg)

(It shows nothing else in details about verification either)

Have I accidentally found a bug?
